src/routers/controllers/removeUserCheckDetailsController.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response } from "express";
+import * as constants from "../../lib/constants";
+import { getTranslationsForView } from "../../lib/utils/translationUtils";
+import { AnyRecord } from "types/utilTypes";
+import { getExtraData, setExtraData } from "../../lib/utils/sessionUtils";
+
+export const removeUserCheckDetailsControllerGet = async (req: Request, res: Response): Promise<void> => {
+    const viewData = getViewData(req);
+
+    res.render(constants.REMOVE_MEMBER_PAGE, { ...viewData });
+};
+
+const getViewData = (req: Request): AnyRecord => {
+    const translations = getTranslationsForView(req.t, constants.REMOVE_MEMBER_PAGE);
+
+    // Hardcoded data will be replaced once relevant API calls available
+    const existingUsers = getExtraData(req.session, constants.MANAGE_USERS_MEMBERSHIP);
+    const companyName = "MORRIS ACCOUNTING LTD";
+    let userToRemove;
+
+    const id = req.params.id;
+
+    for (const i in existingUsers) {
+        if (existingUsers[i].id === id) {
+            userToRemove = existingUsers[i];
```

Comment:
You can either add a [break statement](https://www.typescripttutorial.net/typescript-tutorial/typescript-break/) once you find the user to remove or user use [filter function](https://bobbyhadz.com/blog/typescript-find-object-in-array#find-the-first-object-in-an-array-that-matches-a-condition-in-ts). There is no point to keep iterating over the array if you already have what you were looking for.